src/Altinn.App.Core/Features/TelemetryExtensions.cs

```diff
@@ -0,0 +1,148 @@
+using System.Diagnostics;
+using Altinn.Platform.Storage.Interface.Models;
+using OpenTelemetry.Trace;
+using Labels = Altinn.App.Core.Features.Telemetry.Labels;
+
+namespace Altinn.App.Core.Features;
```

Comment:
consider changing namespace

Reply:
Tried this, improved some places, worsened others. Keeping as is for now